src/components/SimilarStories.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { navigate } from '@reach/router'
+import {
+  EOS_THUMB_UP,
+  EOS_MESSAGE,
+  EOS_CALENDAR_TODAY_FILLED
+} from 'eos-icons-react'
+
+function Card(props) {

```

Comment:
it would be best practice to separate the Card component so that it can be re-used in other components or pages later

Reply:
Done